jwql/website/static

```diff
@@ -0,0 +1 @@
+apps/jwql/static
```

Comment:
Just curious, why did we need to add this to the file?

Reply:
I had to create this in order to run JWQL locally, did I mess up?

Reply:
@penaguerrero you didn't mess up, I think we can remove it. I can take care of it :)